CharacterCreation/03_Feats.txt

```diff
@@ -689,113 +696,182 @@ now working with. This feat can only be taken once per weapon.
-25:	Explosives Expert. -0.5 miss chance @ all ranges (grenades, rocket launchers, etc.)			
-30:	+10% to charisma rolls for intimidation when holding any explosive/launcher			
-40:	Explosives Master. +15% Base damage, Gain feat "Get Down!": Allies in blast radius 
-		may make a DC 60 Reflex save to take half damage (before armor) from friendly fire.			
-		
+	Weapon Expertise feats require proficiency with the weapon type, but otherwise 
+have no prerequisites. Note that as all characters are proficient with Unarmed Strike, 
+Melee Expertise effectively has no prerequisite.
+
+== Pistol Expertise ==	
+
+0:	+5m to ranges R1 & R2. If this would cause R2 to be greater than R3, 
+	also set R3 equal to the new R2.
+12:	May use Weapons - Pistol (Int) in place of Stealth for checks made to prevent 
+	enemies from noticing your Pistol fire.
+24:	Reduce the Reload check penalty imposed by the Dual-Wield / Akimbo feats 
+	by 10 when dual-wielding pistols.
+36:	+1 to Accuracy rolls at all ranges.
+48:	+10m to range R3.
+60:	+10% Base damage. If you succeed on an Accuracy roll by at least 3 or 
+	extra-crit, treat the enemy's armor as having APL 0 for that attack.
+
+== Submachine Gun Expertise ==
+
+0:	+5m to ranges R1 & R2. If this would cause R2 to be greater than R3, 
+	also set R3 equal to the new R2.
+12:	May use Weapons - SMG (Cha) in place of Intimidate for checks to intimidate 
+	a target you are suppressing with an SMG.
+24:	+10 Gun Reflex to one SMG you are wielding.
+36:	+2 to Accuracy rolls at all ranges while firing automatically (not burst).
+48:	+15m to range R3 while firing semi-automatically.
+60:	+10% Base damage. While suppressing an enemy, if you are shot at, after 
+	the first Action of fire you may spend a reaction to duck behind cover 
+	within 5m. This ends your suppression.
+
+== Shotgun Expertise  ==
+
+0:	+5m to range R1. If this would cause R1 to be greater than R2, 
+	also set R2 equal to the new R1.
+12:	May use Weapons - Shotgun (Dex) in place of Acrobatics for checks to execute 
+	a physical maneuver following violent entry of a room.
+24:	If you succeed a reload check by at least 20, reduce the time taken to 
+	reload by one Action, stacking with other reductions (such as from 
+	critical success), to a minimum of one Action.
+36:	Negate the hip-firing Accuracy penalty against enemies within R1 while 
+	firing spreading shot.		
+48:	+10m to range R2. If this would cause R2 to be greater than R3, 
+	also set R3 equal to the new R3.
+60:	+10% Base damage. If you hit armor with a higher APL than your shotgun, 
+	you deal half damage to its AS instead of no damage (and full damage 
+	to health if you deplete the AS).
+
+== Carbine Expertise ==
+
+0:	+10m to range R2. If this would cause R2 to be greater than R3, 
+	also increase R3 to match the new R2.
+
+12:	May use Weapons - Carbine (Dex) in place of Acrobatics on checks to maintain 
+	balance and negate combat penalties associated with a vehicle or mount 
+	you are on board.

```

Comment:
"associated with firing from a moving vehicle or mount"

Reply:
whoops! I missed that in my review.

Reply:
@.@ What horrific cheese monster did I almost create here?